pkg/restore/restore.go

```diff
@@ -176,7 +180,7 @@ func NewKubernetesRestorer(
 		resourceTerminatingTimeout: resourceTerminatingTimeout,
 		resourcePriorities:         resourcePriorities,
 		logger:                     logger,
-		pvRenamer:                  func(string) string { return "velero-clone-" + uuid.NewV4().String() },
+		pvRenamer:                  func(string) string { return "velero-clone-" + veleroCloneUuid.String() },
```

Comment:
I'm pretty sure this change introduces a bug where if multiple PVs are being cloned, either within a single restore or across multiple restores, we'll get a name conflict since we're not generating a new UUID within this function anymore.

Reply:
Ah shoot, good call. So L170 should move into this renamer func?

Reply:
Yeah; it looks like we'll need to add an `error` return parameter to the `pvRenamer` func and handle it appropriately at the call site since the new library may return an error.

Reply:
I'll create an issue for addressing this.

Reply:
#2137

Reply:
:+1: thanks!